.idea/blocksWorld.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please upload only blocksWorld related files. Please remove this one.
This is probably an IDE file.

Please update the .gitignore file to ignore IDE files.